delfin/common/constants.py

```diff
@@ -274,6 +274,15 @@ class InitiatorStatus(object):
     ALL = (ONLINE, OFFLINE, UNKNOWN)
 
 
+class InitiatorType(object):
+    FC = 'fcinitiator'
+    ISCSI = 'iscsiinitiator'
+    NVME_OVER_ROCE = 'roceinitiator'
+    UNKNOWN = 'unknown'
```

Comment:
Can we make type without `initiator`, similar to port type?
```suggestion
    FC = 'fc'
    ISCSI = 'iscsi'
    NVME_OVER_ROCE = 'roce'
    UNKNOWN = 'unknown'
```

Reply:
we can